Makefile

```diff
@@ -99,3 +106,6 @@ endif
 docker:
 	docker build -t "pilosa:$(VERSION)" --build-arg ldflags=$(LDFLAGS) .
 	@echo "Created image: pilosa:$(VERSION)"
+
+docker-build:
+	docker run --rm -v $(PWD):/go/src/$(CLONE_URL) -w /go/src/$(CLONE_URL) -e GOOS=$(GOOS) -e GOARCH=$(GOARCH) golang:latest go build -ldflags $(LDFLAGS) $(FLAGS) $(CLONE_URL)/cmd/pilosa
```

Comment:
Does the Go version used to create the build get stored anywhere? pinning this to golang:latest means it isn't actually repeatable since that's a moving target.